x-pack/protocol/src/main/java/org/elasticsearch/protocol/xpack/ml/datafeed/Datafeed.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.protocol.xpack.ml.datafeed;
+
+import org.elasticsearch.common.ParseField;
+
+/**
+ * Collection of common fields for Datafeed objects
+ */
+public final class Datafeed {
```

Comment:
I do not see the need for having this in its own class. For the client, `job_id` is a datafeed field so it might as well just live in `DatafeedConfig` with the rest of them.